ARCHITECTURE.md

```diff
@@ -126,10 +204,34 @@ We first need to clone the repo:
 git clone https://github.com/CoopHive/eth-faucet
 ```
 
-#### install stack
+#### clone coophive repo
+
+We first need to clone the repo:
 
 ```bash
+git clone git@github.com:CoopHive/coophive.git
 cd coophive
+```
+
+In order for the `stack` script command to successfully run on mac arm64, `LOCAL_GETH` flag needs to be set.
+
+```bash
+vim stack
```

Comment:
better to set it in the .env file and the user’s environment otherwise this will surely get accidentally committed by the developer